code/API_definitions/qod-api.yaml

```diff
@@ -329,16 +332,32 @@ components:
       type: object
       required:
         - sessionId
-        - event
+        - eventReport
       properties:
         sessionId:
           $ref: "#/components/schemas/SessionId"
-        event:
+        eventReport:
           $ref: "#/components/schemas/SessionEvent"
     SessionEvent:
-      type: string
-      enum:
-        - SESSION_TERMINATED
+      type: object
+      required:
+        - event
+      properties:
+        event:
+          type: string
+          enum:
+            - QOS_STATUS_CHANGED
+        qosStatus:
```

Comment:
It's little confusing when qosStatus also contains 'reason', a different name like eventInfo would be clearer.